content/using_ec2_spot_instances_with_eks/cleanup.md

```diff
@@ -21,9 +21,22 @@ kubectl delete -f monte-carlo-pi-service.yml
 helm delete kube-ops-view metrics-server
 ```
 
-## Removing eks nodegroups
-```
+## Removing Spot node groups
+```bash
+eksctl delete nodegroup -f spot_nodegroup_4vcpu_16gb.yml --approve
```

Comment:
Given that the self managed nodegroup is optional, perhaps we can add a section and something that says, if you've followed the optional path, delete the relevant groups with this command.

Reply:
Moved the Self managed Spot node group cleanup to a different section and mentioned that it is (optional).